### PR TITLE
MCLOUD-3908: Static improvements to php.ini

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-3908

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: php.ini [hunk 1]
```diff
@@ -45,3 +40,7 @@ session.gc_probability = 1
 ;
 opcache.validate_timestamps = 0
 opcache.blacklist_filename="${MAGENTO_CLOUD_APP_DIR}/op-exclude.txt"
+opcache.memory_consumption=512
```
Comment (kandy) [inline php.ini:43]: opcache.memory_consumption=256  << this should be 64 on integration
opcache.max_accelerated_files=16229

Comment (shiftedreality) [reply php.ini:43]: Shouldn't opcache.max_accelerated_file be >= # of Magento files?

Comment (shiftedreality) [reply php.ini]: `opcache.memory_consumption` is 128 by default https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.memory-consumption

Comment (kandy) [reply php.ini]: > Shouldn't opcache.max_accelerated_file be >= # of Magento files
No, only used files

Do we have possibility to track target branch on build service? We should generate php.ini memory_consumption based on type of environment

Comment (kandy) [reply php.ini]: Also you can use https://github.com/rlerdorf/opcache-status to get statistics

**Comment by andriyShevtsov** [comment]
QA passed

**Pull request merged**